rkt.spec

```diff
@@ -97,10 +120,24 @@ install -dp -m 700 %{buildroot}%{_sharedstatedir}/%{name}/containers
 %{_unitdir}/%{name}-metadata.socket
 %{_unitdir}/%{name}-metadata.service
 
-%{_sharedstatedir}/%{name}/cas
-%{_sharedstatedir}/%{name}/cas/db
-%{_sharedstatedir}/%{name}/tmp
-%{_sharedstatedir}/%{name}/containers
+%attr(2770, root, rkt) %{_sharedstatedir}/%{name}
```

Comment:
It should be possible.

I will let RPM packagers decide whether they prefer to set permissions in the RPM-way (and synchronize with upstream if the directories change) or if they prefer to exec `rkt install`.